src/Analyzer.Utilities/JsonLineNumberResolver.cs

```diff
@@ -14,7 +14,8 @@ namespace Microsoft.Azure.Templates.Analyzer.Utilities
     /// </summary>
     public class JsonLineNumberResolver : ILineNumberResolver
     {
-        private static readonly Regex resourceIndexInPath = new Regex(@"resources\[(?<index>\d+)\]", RegexOptions.IgnoreCase | RegexOptions.Compiled);
+        private static string resourceIndexPattern = @"resources\[(?<index>\d+)\]";
+        private static readonly Regex resourceIndexInPath = new Regex(resourceIndexPattern, RegexOptions.IgnoreCase | RegexOptions.Compiled);
```

Comment:
This change can be reverted so the string is specified directly in the Regex creation.